examples/semi_supervised/plot_label_propagation_digits.py

```diff
@@ -69,15 +73,15 @@ class will be very good.
 print("Confusion matrix")
```

Comment:
Isolate the `classification_report` in its own cell. You can remove the `print` around the function since it will be displayed.